tests/ha/single_pod_failure/test_data_pod_failure.py

```diff
@@ -69,8 +67,6 @@ def setup_class(cls):
         Setup operations for the test file.
         """
         LOGGER.info("STARTED: Setup Module operations.")
-        cls.csm_user = CMN_CFG["csm"]["csm_admin_user"]["username"]
-        cls.csm_passwd = CMN_CFG["csm"]["csm_admin_user"]["password"]
         cls.num_nodes = len(CMN_CFG["nodes"])
         cls.username = []
```

Comment:
just make this as list(), check at all places

Reply:
Not seeing any codacy warning and not even getting any issues in test execution, so will keep it as is.